src/server/pfs/server/url_worker.go

```diff
@@ -176,7 +175,6 @@ func (d *driver) processGetFileURLTask(ctx context.Context, task *GetFileURLTask
 }
 
 func serializePutFileURLTask(task *PutFileURLTask) (*types.Any, error) {
-	fmt.Printf("task: %+v\n", task)
```

Comment:
Not sure if you forgot this while debugging.  Remove or, if you want a log line, use our internal `log` package.

Reply:
Yeah I think this is part of PFS-25, so I will remove that from there and this diff should vanish.